### PR TITLE
update to readme with delete vm commands

### LINKED ISSUE #13: Add delete and purge command to README
Find the command to delete and purge multipass of VMs.
Also find the command to delete multipass altogether.
Add these commands to the README

### PR DESCRIPTION
**Pull request by DevinBeu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,9 +76,17 @@ main branch.
...
+# PROJECT COMMANDS:
 
 To start the project on a Mac run:
 bash installVM-mac.sh && multipass shell mos-vm
 
+To delete all VM's created with multipass run:
+multipass delete --all --purge
+
+To remove multipass from your linuxmint machine run:
+sudo snap remove multipass
```
Comment (stephenlauck) [inline README.md:88]: You don't need to uninstall multipass, maybe a bit too much.

Comment (Topher2014) [reply README.md:88]: We just put this in here to initially test that multipass installs when not already installed.

**Review by MonetCarriere — APPROVED** [review]
The multipass delete --all --purge command works! Great job. 😎
Push it on up to main 🎉

**Comment by DevinBeu** [comment]
Both the `delete multipass` and `install multipass` command have been tested, both of them are working.

**Review by Topher2014 — APPROVED** [review]

**Pull request merged**